src/app/app.component.ts

```diff
@@ -1,16 +1,22 @@
-import { Component } from '@angular/core';
-import { RouterModule } from '@angular/router';
-import { HomeComponent } from '../components/home/home.component';
-import { HeaderComponent } from '../components/header/header.component';
-import { FooterComponent } from '../components/footer/footer.component';
+import { Component } from "@angular/core";
+import { Router, RouterModule } from "@angular/router";
+import { HeaderComponent } from "./shared/header/header.component";
+import { HomeComponent } from "../components/home/home.component";
+import { FooterComponent } from "../components/footer/footer.component";
 
 @Component({
   standalone: true,
   imports: [HomeComponent, RouterModule, HeaderComponent, FooterComponent],
-  selector: 'pet-paradise-client-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  selector: "pet-paradise-client-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.scss"],
 })
```

Comment:
we were using the simple cuotes on all the components for all this section, please change it